spec/views/posts_index_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe 'post_index', type: :feature do
```

Comment:
- [ ] Good job so far with your tests 👍🏽 .Kindly add tests that assert you can see how many likes a post has as specified in the project requirement